app/utils/validator.js

```diff
@@ -0,0 +1,12 @@
+export const validator = (value, words, length = 1) => {
```

Comment:
What is the use of `length` here?

Reply:
There is some validation on backend for that purpose I used length too with default value.

Reply:
Can you please point me to that ?

Reply:
This is the validation done on the backend API
![image](https://user-images.githubusercontent.com/85296770/216375196-fac960af-f1be-4bba-a7e7-f78e397bea69.png)